src/components/cylc/tree/TreeItem.vue

```diff
@@ -257,15 +253,8 @@ export default {
     },
   },
 
-  emits: [
-    ...passthroughEvents
-  ],
-
   data () {
     return {
-      active: false,
-      selected: false,
-      filtered: true,
```

Comment:
FYI: The intention for this component was to support [multiple] selection, hence these (presumably unused) attributes.

We can presumably use the same weakmap for these when we get around to it later?

Reply:
Yes I suppose so